.gitignore

```diff
@@ -0,0 +1,6 @@
+*.py[cod]
+*$py.class
+configuration.py
+!configuration.py.example
+personal/
+jython-standalone-2.7.0.jar
```

Comment:
What if we add the /automation/jython/ directory to the repo, and just ignore its contents?
```
Core/automation/jython/**
```

Reply:
Way better! ;)